docker-process.el

```diff
@@ -53,6 +57,11 @@
       (message command)
       (shell-command-to-string command))))
 
+(defun docker-use-format (go-template)
+  "If docker-command-podman is set return json else the given go template"
+  (if (string-match-p "podman" docker-command)  "json"  go-template)
+  )
```

Comment:
Don't put closing parenthesis on their own in elisp. See https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html